cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/omri/Programs/ns-3/workspace/ns-allinone-3.14.1/ns-3.14.1/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Tools/c_config.py", line 445, in run_c_code\n    bld.compile()\n  File "/home/omri/Programs/ns-3/workspace/ns-allinone-3.14.1/ns-3.14.1/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Build.py", line 190, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 140601505113296: cxxprogram test.cpp.1.o -> testprog}\n[\'/usr/bin/g++\', \'test.cpp.1.o\', \'-o\', \'/home/omri/Programs/ns-3/workspace/ns-allinone-3.14.1/ns-3.14.1/build/.conf_check_39b3cecfac3989b2ed760243cef109e6/testbuild/testprog\', \'-Wl,-Bstatic\', \'-Wl,-Bdynamic\', \'-lsqlite3\']\n'
